dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -449,9 +451,22 @@ public void publish(List<String> args) {
             publishHelp();
         } else {
             String first = reqVal(args, "--entry");
-            String entryname = optVal(args, "--entryname", null);
             final boolean unpublishRequest = args.contains("--unpub");
-            handlePublishUnpublish(first, entryname, unpublishRequest);
+
+            // --newEntryName is the desired parameter flag, but maintaining backwards compatibility for --entryname
+            String newEntryName = optVal(args, "--newEntryName", null);
```

Comment:
Unless we already do this elsewhere, I don't think we add mixed case parameter names to the command-line.

Reply:
Looks like dash-delimited is the most common, `word-word-word`. I'll switch to that pattern. Thanks!